tests/flow/test_function_calls.py

```diff
@@ -2104,7 +2085,9 @@ def test83_Replace(self):
             "RETURN replace('abcabc', 'bc', '0')": [["a0a0"]],
             "RETURN replace('abcabc', 'abc', '')": [[""]],
             "RETURN replace('abcabc', 'ab', '')": [["cc"]],
-            "RETURN replace('abcabc', '', '0')": [["0a0b0c0a0b0c0"]]
+            "RETURN replace('abcabc', '', '0')": [["0a0b0c0a0b0c0"]],
+            # test unicode charecters
+            "RETURN replace('丁丂七丄丅丆万丈三上', '\xe4', 'X')": [["丁丂七丄丅丆万丈三上"]],
```

Comment:
we're not expecting to see a change in the string, please add a comment explaining this and why.